docs-src/concepts/what-is-a-retention-period.md

```diff
@@ -26,4 +26,6 @@ Setting the Retention Period to 0 results in the error _A valid retention period
 If you don't set the Retention Period value when using the [`tctl namespace register`](/tctl-v1/namespace#register) command, it defaults to 3 days.
 If you don't set the Retention Period value when using the Register Namespace Request API, it returns an error.
 
+When updating the Retention Period, the new duration only applies to the workflows that close after the update.
```

Comment:
```suggestion
When changing the Retention Period, the new duration applies to Workflow Executions that close after the change is saved.
```

Reply:
Good pickup @flossypurse. I agree "Workflow Execution" makes it a lot clearer.